BuildResidentialHPXML/measure.rb

```diff
@@ -951,12 +951,14 @@ def arguments(model)
     heating_system_type_choices << HPXML::HVACTypeFurnace
     heating_system_type_choices << HPXML::HVACTypeWallFurnace
     heating_system_type_choices << HPXML::HVACTypeFloorFurnace
-    heating_system_type_choices << HPXML::HVACTypeBoiler
+    heating_system_type_choices << "#{HPXML::HVACTypeBoiler}, In-Unit, w/ Baseboard"
     heating_system_type_choices << HPXML::HVACTypeElectricResistance
     heating_system_type_choices << HPXML::HVACTypeStove
     heating_system_type_choices << HPXML::HVACTypePortableHeater
     heating_system_type_choices << HPXML::HVACTypeFireplace
     heating_system_type_choices << HPXML::HVACTypeFixedHeater
+    heating_system_type_choices << "#{HPXML::HVACTypeBoiler}, Shared, w/ Baseboard"
+    heating_system_type_choices << "#{HPXML::HVACTypeBoiler}, Shared, w/ Ductless Fan Coil"
```

Comment:
... and then here, we could simplify to `"Shared #{HPXML::HVACTypeBoiler} w/ Baseboard"` and `"Shared #{HPXML::HVACTypeBoiler} w/ Ductless Fan Coil"`.